data/analytics/hooks.ts

```diff
@@ -10,6 +10,12 @@ export type AnalyticsQueryParams = {
   interval?: "minute" | "hour" | "day" | "week" | "month";
 };
 
+// TODO: Keep updated with actual ClickHouse data
+export const SUPPORTED_ANALYTICS_CHAINS = [
```

Comment:
would be nice to do Ethereum.ChainId etc instead of the actual number for better redeability